ocis-phoenix/pkg/service/v0/service.go

```diff
@@ -168,9 +169,9 @@ func (p Phoenix) Static(ttl int) http.HandlerFunc {
 			return
 		}
 
-		w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%s", strconv.Itoa(ttl)))
+		w.Header().Set("Cache-Control", fmt.Sprintf("max-age=%s, must-revalidate", strconv.Itoa(ttl)))
+		w.Header().Set("Expires", expires)
```

Comment:
From MDN `If there is a Cache-Control header with the max-age or s-maxage directive in the response, the Expires header is ignored.` Do we need to specify it then?

Reply:
There is a middleware in place on every service, that introduces an expiration date (set to `epoch`). So we need to either remove it from the header again or set a valid expiration date. I prefer to have the latter, but having the `Del` call on the header should do the same. 🤷

Reply:
I see. Thanks!